config/field_test.yml

```diff
@@ -9,6 +9,30 @@ experiments:
       - user_creates_reaction
       - user_views_article_four_days_in_week
       - user_views_article_four_hours_in_day
+  follow_implicit_points: # Points given for implicit tag weights
+    # Implemented here: app/workers/follows/update_points_worker.rb
+    # Hypotheses: Weighting tags based on implied preference will lead
+    # to more repeat engagement.
+    # Recommended duration: 4 weeks
```

Comment:
❤️  Maybe put the date we are starting this so we don't have to go to the git blame to check when this comment was added.